src/renderers/WebGLRenderer.js

```diff
@@ -1196,12 +1196,13 @@ function WebGLRenderer( parameters = {} ) {
 	function renderTransmissionPass( opaqueObjects, scene, camera ) {
 
 		const isWebGL2 = capabilities.isWebGL2;
+		const hasHalfFloatColorBuffer = extensions.has( 'EXT_color_buffer_half_float' ) || ( isWebGL2 && extensions.has( 'EXT_color_buffer_float' ) );
```

Comment:
Isn't it sufficient to just test for `EXT_color_buffer_half_float`? 

I don't think there is a WebGL 2 device that does `EXT_color_buffer_float` but not `EXT_color_buffer_half_float`.

Reply:
Yes no reasonable device would do that, but the `EXT_color_buffer_float` [specification](https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_float/) says `This is a difference in functionality compared to the [EXT_color_buffer_half_float](https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_half_float) extension`, so I guess a real evil guy can do it.

Reply:
I think I would take this risk and go for simplicity. Meaning only checking for `EXT_color_buffer_half_float`.

Reply:
Commit updated. Please note that https://github.com/mrdoob/three.js/blob/8db7a598ed6e71538e286e5c56c02cc5823157dc/src/renderers/WebGLRenderer.js#L1976 is also doing the `EXT_color_buffer_float` check, so there's a little inconsistency.

Reply:
If the approach of this PR works fine, we can refactor this code with a later PR.